### PR TITLE
fix link to weekly meeting

### PR DESCRIPTION
**Pull request by dholbach**
Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

File: sig-list.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 # SIGs and Working Groups
 
 Most community activity is organized into Special Interest Groups (SIGs),
-time bounded Working Groups, and the [community meeting](communication.md#weekly-meeting).
+time bounded Working Groups, and the [community meeting](communication/README.md#weekly-meeting).
```
Comment (nikhita) [inline sig-list.md]: @dholbach Good catch!

This is generated from https://github.com/kubernetes/community/blob/2f87761d8aa7043946a10ef7123ce1d78a4096cd/generator/list.tmpl#L5. Could you make the change here?

Comment (dholbach) [reply sig-list.md]: Sure.

Comment (nikhita) [reply sig-list.md]: And then run `make` for generation :)

**Comment by nikhita** [comment]
Lgtm, could you squash the commits into a single commit?

Also, a small nit, can you remove the github username from the commit message? GitHub sends emails whenever the commit is picked into someone's fork. :see_no_evil:

**Comment by dholbach** [comment]
> Lgtm, could you squash the commits into a single commit?

Will do.

> Also, a small nit, can you remove the github username from the commit message? GitHub sends emails whenever the commit is picked into someone's fork. see_no_evil

Hah, ok. :-)

File: sig-list.md [hunk 2]
```diff
@@ -67,6 +67,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
...
 |[Resource Management](wg-resource-management/README.md)||* [Vishnu Kannan](https://github.com/vishh), Google<br>* [Derek Carr](https://github.com/derekwaynecarr), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/wg-resource-mgmt)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-wg-resource-management)|* Regular WG Meeting: [Wednesdays at 11:00 PT (Pacific Time) (biweekly (On demand))](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
 |[Security Audit](wg-security-audit/README.md)||* [Aaron Small](https://github.com/aasmall), Google<br>* [Joel Smith](https://github.com/joelsmith), Red Hat<br>* [Craig Ingram](https://github.com/cji), Salesforce<br>* [Jay Beale](https://github.com/jaybeale), InGuardians<br>|* [Slack](https://kubernetes.slack.com/messages/wg-security-audit)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-wg-security-audit)|* Regular WG Meeting: [Mondays at 13:00 PT (Pacific Time) (weekly)](https://docs.google.com/document/d/1RbC4SBZBlKth7IjYv_NaEpnmLGwMJ0ElpUOmsG-bdRA/edit)<br>
+
```
Comment (nikhita) [inline sig-list.md]: oops, looks unrelated, can you remove this?

Comment (dholbach) [reply sig-list.md]: This was introduced by running `make`.

Comment (nikhita) [reply sig-list.md]: oh right, this is because of the newline. lgtm.
